Base/Logic/Resources/ColorFiles/DarkBrightChartColors.txt

```diff
@@ -4,7 +4,7 @@
 #Filename: Slicer4_2012_DarkBrightChartColors.txt
 #Title: Slicer4:2012 DarkBrightChartColors
 #PageName: Slicer4:2012_DarkBrightChartColors
-#Source: http://www.slicer.org/slicerWiki/index.php/Slicer4:2012_DarkBrightChartColors
+#Source: https://www.slicer.org/slicerWiki/index.php/Slicer4:2012_DarkBrightChartColors
```

Comment:
@lassoan  Similarly to svg/markups/... files, should we consider occurrence of such URL in color file as a schema reference ?

Reply:
After further consideration, there are URL and I think we should be updated. I will create an issue to track the fact we need to find these a more legitimate home outside of the wiki.

Reply:
For reference, here is the issue https://github.com/Slicer/Slicer/issues/6305